docs/runbooks/DeprecatedRHEL6Vm.md

```diff
@@ -0,0 +1,22 @@
+# DeprecatedRHEL6Vm
+
+## Meaning
+
+This alert fires when a running virtual machine is based on RHEL6 template, which will not be supported in the next release.
+
+## Impact
+RHEL6 virtual machines will not be supported in the next release in case of issues.
+
+## Diagnosis
+
+1. Check which running virtual machines has the rhel6 template label.
+
+   ```bash
+   $ kubectl get -A vm -ojson | jq  '.items | map(select((.metadata.labels."vm.kubevirt.io/template" | test("rhel6-.*")?) and .status.printableStatus == "Running")) | {kind: "List", items: .}'  | kubectl get -A -f -
+   ```
+
+## Mitigation
+
+Upgrade all the RHEL6 virtual machines to a newer version or stop them.
+<!--DS: If you cannot resolve the issue, log in to the link:https://access.redhat.com[Customer Portal] and open a support case, attaching the artifacts gathered during the Diagnosis procedure.-->
```

Comment:
@fossedihelm 
We don't need this section here. There is a different one for upstream.

Reply:
If you cannot resolve the issue, see the following resources:

OKD Help

#virtualization Slack channel

No need for the US commenting.